### DISCUSSION TITLE
Welcome to OneSTools.EventLog Discussions!

### DISCUSSION THREAD

**Discussion by akpaevj** [akpaevj/OneSTools.EventLog#10] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kuzja086** [comment]
Здравствуйте, при выгрузке в ClickHouse выгрузилось примерно 5 млн записей. Настройки все по умолчанию, оперативки забирало примерно 100 мб. А далее начинается рост потребления памяти и в итоге приложение завершается с ошибкой, что недостаточно памяти. Есть предположения в чем может быть причина или в какую сторону копать?

**Reply by akpaevj** [reply]
Здравствуйте. Такое может быть из-за ошибки парсинга данных журнала. Найдите файл и позицию последнего обработанного события. Нужно посмотреть листинг следующего за ним события, скорее всего в нем проблема.

**Reply by akpaevj** [reply]
> Здравствуйте, при выгрузке в ClickHouse выгрузилось примерно 5 млн записей. Настройки все по умолчанию, оперативки забирало примерно 100 мб. А далее начинается рост потребления памяти и в итоге приложение завершается с ошибкой, что недостаточно памяти. Есть предположения в чем может быть причина или в какую сторону копать?

Было бы хорошо, если бы Вы смогли приложить файл сбойного журнала. Тогда бы я смог понять что и где идёт не так, все остальное - гадание на кофейной гуще

**Reply by kuzja086** [reply]
У меня пока была песочница, я прибил журнал, сделал новый он выгружался, потом в какой-то момент просто подвис без ошибок. Начал читать с какой-то позиции и все. Затем я обновился до актуальной версии, перезапустил и все работает отлично. Пока проблем не наблюдаю, Спасибо

**Comment by vadimaverin** [comment]
Здравствуйте. При попытке запустить службу получаю ошибку [SC] StartService: ошибка: 1053: Служба не ответила на запрос своевременно.
ОС Windows 10. runtime .net 5 установлен. Onescript последней версии установлен. Подскажите в чем проблема может быть ?

**Reply by vadimaverin** [reply]
Как службу запускаю "C:\EventLogExporter\EventLogExporter.exe"

**Reply by akpaevj** [reply]
А причём здесь onescript то?) Обновите runtime dotneta до 5 версии и все

**Reply by akpaevj** [reply]
И опишите проблему по шаблону в issues

**Comment by w82hands** [comment]
У меня все работает и нет ошибок, но есть один момент, когда в live режиме работает EventLogExportersManager то 1с не показывает последние события, нет уверенности и в том, что она их запишет, т.к. в тестах теряли записи. 
 "Exporter": {
    "StorageType": 1,
    "Portion": 10000,
    "TimeZone": "Asia/Novosibirsk",
    "WritingMaxDegreeOfParallelism": 4,
    "CollectedFactor": 4,
    "ReadingTimeout": 2,
    "LoadArchive": false,
    "SkipEventsBeforeDate": "2020-05-01T00:00:00"

Мб, увеличить таймаут или дайте совет? 
настройки следующие. 

Также еще есть смещение во времени на 7 часов, хотя пояса везде установлены верно, в клике, в системе и в redash, которым смотрю лог.

**Comment by AlekseyUAM** [comment]
Здравствуйте,
У меня зависает при чтении текущего файла журнала регистрации. При этом блокирует файл и 1С перестает в него писать.

collector_1   | info: OneSTools.EventLog.Exporter.Core.EventLogExporter[0]
collector_1   |       File 20241201000000.lgp will be read from 1520644923 position, LGF file will be read from 131934 position

**Comment by killallxxx** [comment]
Добрый день! Подскажите пожалуйста, какого размера будет примерно база данных кликхаус при выгрузке в неё журнала регистрации объемом от 500мб до 4гб ежедневно? Через неделю, месяц, год.

**Comment by killallxxx** [comment]
Подскажите пожалуйста! Умеет ли сервис использовать TLS?

**Comment by w82hands** [comment]
У меня час весит до 6 гигов бывает))) в lgp, а вот 100 гигов весит в клике база за 1 год и 2 месяца  10 миллиардов строк. У меня так вышло.ерп. Она в активной разработке, поэтому так. А вот ух 44 миллиарда строк 3 года +, 32 гигабайта.10 окт. 2025 г. 19:15 пользователь killallxxx ***@***.***> написал:
Добрый день! Подскажите пожалуйста, какого размера будет примерно база данных кликхаус при выгрузке в неё журнала регистрации объемом от 500мб до 4гб ежедневно? Через неделю, месяц, год.

**Comment by w82hands** [comment]
Ему он не нужен, ну а сам клик+ какой- то вихуализатор, пожалуйста, всё, что душе угодно. Я внутри сети всё использую, не заморачивался пока иб не сказали. 10 окт. 2025 г. 19:23 пользователь killallxxx ***@***.***> написал:
Подскажите пожалуйста! Умеет ли сервис использовать TLS?

**Comment by w82hands** [comment]
Надеюсь, что понятно описываю, резюмирая, можно сказать, что в разных конфигурациях журнал весит по разному как файлово, так и в клике после экспортера. Нас объем устраиваеь в итоге. Несколько баз с 4 тб файлово в журнале, весят суммарно 400 гигов в клике... как - то так10 окт. 2025 г. 19:15 пользователь killallxxx ***@***.***> написал:
Добрый день! Подскажите пожалуйста, какого размера будет примерно база данных кликхаус при выгрузке в неё журнала регистрации объемом от 500мб до 4гб ежедневно? Через неделю, месяц, год.
